client/my-sites/site-settings/search.jsx

```diff
@@ -44,20 +43,17 @@ class Search extends Component {
 		fields: PropTypes.object,
 	};
 
-	renderInfoLink( url ) {
+	renderInfoLink( link, privacyLink ) {
```

Comment:
Is this needed?
As far as I can see, `renderInfoLink` is only used once later, with only one argument.

Reply:
Apologies. I didn't see this particular comment until I had already merged batch 1, so I continued onward through merging the rest of these so they'd go in together. They're all in now. Huge props for your help!

Regarding `renderInfoLink()`. I see that you're right. However, this does make it easier for the call down below to `renderInfoLink()` to be updated in the future; i.e., in case the privacy information link changes, it can be passed in separately.

If you feel strongly about removing this, I'm happy to do that in a follow-up.